webhook_server/app.py

```diff
@@ -21,7 +21,8 @@ def webhook():
     #     abort(403)
     
     for userid in json.loads(request.data)['list_folder']['accounts']:
-        if userid not in user_queues:
+        userid = userid.split(':')[1]
```

Comment:
Same here, if the user id does not contain ":". The list will only have a length of 1.